core/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -282,6 +295,12 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeOptionalStreamable(breaker);
         out.writeOptionalStreamable(scriptStats);
         out.writeOptionalStreamable(discoveryStats);
+        if (ingestStats == null) {
```

Comment:
Yes, I meant that we should add this method which does not exist.

It could be done in another PR.